index.bs

```diff
@@ -231,14 +231,48 @@ Each {{console}} namespace object has an associated <dfn>timer table</dfn>, whic
 
 <h4 id="timeend" oldids="timeend-label,dom-console-timeend" method for="console">timeEnd(|label|)</h4>
 
-1. Let |startTime| be the result of [=map/getting=] the value of the entry with key |label| in the
-   associated <a>timer table</a>.
+1. Let |timerTable| be the associated <a>timer table</a>.
+1. Let |startTime| be |timerTable|[|label|].
+1. [=map/set|Remove=] |timerTable|[|label|].
```

Comment:
This should probably `map/remove` but I have to take a closer look

Reply:
You should be able to just do `[=map/Remove=]`